kyaml/openapi/openapi_benchmark_test.go

```diff
@@ -55,10 +55,10 @@ func BenchmarkOpenAPIV2ParseDocument(t *testing.B) {
 func BenchmarkProtoUnmarshal(t *testing.B) {
 	assetName := filepath.Join(
 		"kubernetesapi",
-		"v1218pb",
+		"v1218",
 		"swagger.pb")
 
-	b := v1218pb.MustAsset(assetName)
+	b := v1218.MustAsset(assetName)
```

Comment:
Is this benchmark suite still valid without the json committed? I see `BenchmarkAssetRead` below is expecting `swagger.json` to still exist

Reply:
That's a good point, I'll remove all the ones that are expecting a `swagger.json`.